website/content/docs/templates/hcl_templates/variables.mdx

```diff
@@ -79,33 +79,29 @@ name for the variable, which must be unique among all variables in the same
 build. This name is used to assign a value to the variable from outside and to
 reference the variable's value from within the build.
 
-The `variable` block can optionally include a `type` argument to specify what
-value types are accepted for the variable, as described in the following
-section.
 
-The `variable` declaration can also include a `default` argument. If present,
-the variable is considered to be _optional_ and the default value will be used
-if no value is set when calling the build or running Packer. The `default`
-argument requires a literal value and cannot reference other objects in the
-configuration.
+## Arguments
 
-## Using Input Variable Values
+Terraform CLI defines the following optional arguments for variable declarations:
```

Comment:
I'm confused, why does this page mention Terraform?

Reply:
Oops. Fixing now.

Reply:
For context - it was copy paste, I liked the arguments section in Terraform so I replicated on the Packer docs 😄